quipucords/api/fingerprint/tests_fingerprint_model.py

```diff
@@ -38,6 +38,24 @@ def test_empty_fingerprint(self):
         self.assertTrue(is_valid)
         serializer.save()
 
+    # pylint: disable=invalid-name
+    def test_product_with_version_fingerprint(self):
+        """Create a fingerprint with products."""
+        product_dict = {'name': 'product1',
+                        'presence': 'unknown',
+                        'version': ['1', '2'],
+                        'metadata': {}}
+        fingerprint_dict = {'report_id': self.fact_collection.id,
+                            'metadata': {},
+                            'products': [product_dict]}
+
+        serializer = FingerprintSerializer(data=fingerprint_dict)
+        is_valid = serializer.is_valid()
+        if not is_valid:
+            print(serializer.errors)
```

Comment:
Should this print be here?

Reply:
Lol. You put it there original. I've just been copying it since then. Its useful for debug if you break the tests, but otherwise doesn't get hit.

Reply:
I think its awesome code.